### ISSUE TITLE
`rm_terraform` function erases `.terraform-docs.yaml` files

### ISSUE THREAD

**Issue by drewmullen** [antonbabenko/pre-commit-terraform#344]
### Describe the bug

the example for `rm_terraform` functions and deletes terraform-docs files:

```shell
find . -name ".terraform*" -print0
./.terraform./.terraform/modules/tags/.terraform-docs.yaml.
```

### How can we reproduce it?

see above

### Environment information

* OS:  

* `uname -a` and/or `systeminfo | Select-String "^OS"` output:

```bash
$ uname -a
Darwin 3c22fb40eee8 19.6.0 Darwin Kernel Version 19.6.0: Thu Jan 13 01:26:33 PST 2022; root:xnu-6153.141.51~3/RELEASE_X86_64 x86_64
```

* Tools availability and versions:

```bash
INSERT_TOOLS_VERSIONS_HERE
```

* `.pre-commit-config.yaml`:

<details><summary>file content</summary>

```bash
INSERT_FILE_CONTENT_HERE
```

</details>

**Comment by antonbabenko** [comment]
Thank you very much, Drew!

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.64.1 :tada:
